### ISSUE TITLE
Build fails - powerdns no longer has packages for Debian jessie

### ISSUE THREAD

**Issue by xandm** [pboehm/ddns#21]
The following errors appear, it appears there are no longer Debian jessie packages for powerdns:

```
The following extra packages will be installed:
  init-system-helpers libboost-program-options1.55.0 libluajit-5.1-2
  libluajit-5.1-common libsodium13
Suggested packages:
  pdns-backend
The following NEW packages will be installed:
  init-system-helpers libboost-program-options1.55.0 libluajit-5.1-2
  libluajit-5.1-common libsodium13 pdns-backend-remote pdns-server
0 upgraded, 7 newly installed, 0 to remove and 0 not upgraded.
Need to get 2454 kB of archives.
After this operation, 11.2 MB of additional disk space will be used.
Get:1 http://deb.debian.org/debian/ jessie/main libboost-program-options1.55.0 amd64 1.55.0+dfsg-3 [143 kB]
Get:2 http://deb.debian.org/debian/ jessie/main libluajit-5.1-common all 2.0.3+dfsg-3 [36.6 kB]
Get:3 http://deb.debian.org/debian/ jessie/main libluajit-5.1-2 amd64 2.0.3+dfsg-3 [204 kB]
Err http://repo.powerdns.com/debian/ jessie-auth-41/main pdns-server amd64 4.1.13-1pdns.jessie
  404  Not Found
Err http://repo.powerdns.com/debian/ jessie-auth-41/main pdns-backend-remote amd64 4.1.13-1pdns.jessie
  404  Not Found
Get:4 http://deb.debian.org/debian/ jessie/main libsodium13 amd64 1.0.0-1 [138 kB]
Get:5 http://deb.debian.org/debian/ jessie/main init-system-helpers all 1.22 [14.0 kB]
Fetched 536 kB in 0s (16.7 MB/s)
E: Failed to fetch http://repo.powerdns.com/debian/pool/main/p/pdns/pdns-server_4.1.13-1pdns.jessie_amd64.deb  404  Not Found

E: Failed to fetch http://repo.powerdns.com/debian/pool/main/p/pdns/pdns-backend-remote_4.1.13-1pdns.jessie_amd64.deb  404  Not Found

E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
ERROR: Service 'powerdns' failed to build: The command '/bin/sh -c echo "deb http://repo.powerdns.com/debian jessie-auth-41 main" > /etc/apt/sources.list.d/pdns.list 	&& echo "Package: pdns-*\nPin: origin repo.powerdns.com\nPin-Priority: 600\n" >> /etc/apt/preferences.d/pdns 	&& curl https://repo.powerdns.com/FD380FBB-pub.asc | apt-key add - 	&& apt-get -y update 	&& apt-get install -y pdns-server pdns-backend-remote 	&& rm -rf /var/lib/apt/lists/*' returned a non-zero code: 100
```

**Comment by pboehm** [comment]
Hello,

could you try replacing the `FROM buildpack-deps:jessie-scm` with `FROM buildpack-deps:buster-scm` in `docker/powerdns/Dockerfile`? I hope that this solves the problem.

**Comment by xandm** [comment]
I've replaced all references to `jessie` with `buster` in `docker/powerdns/Dockerfile` and build now succeeds. However the powerdns container will continuously restart. I'm afraid I don't have the Docker/PowerDNS knowledge to troubleshoot this.

```
CONTAINER ID        IMAGE                   COMMAND                  CREATED             STATUS                          PORTS                                                NAMES
4fa860e3a2be        abiosoft/caddy:latest   "/bin/parent caddy -…"   41 minutes ago      Up 41 minutes                   0.0.0.0:80->80/tcp, 0.0.0.0:443->443/tcp, 2015/tcp   ddns_caddy_1
b94416a2f889        ddns_powerdns           "/entrypoint.sh pdns…"   41 minutes ago      Restarting (1) 36 minutes ago                                                        ddns_powerdns_1
c84adc89cbd2        ddns_ddns               "/bin/sh -c '/go/bin…"   41 minutes ago      Up 41 minutes                                                                        ddns_ddns_1
93db97f3b18c        redis:4-alpine          "docker-entrypoint.s…"   About an hour ago   Up About an hour                6379/tcp                                             ddns_redis_1
```

**Comment by pboehm** [comment]
I will have a look into this later in the evening.

**Comment by bslavin** [comment]
I can confirm and am having a similar issue with the restarts.

**Comment by pboehm** [comment]
@xandm @bslavin I've fixed the PowerDNS build by upgrading to PowerDNS 4.3 and use stretch as base image while also removing a now unsupported config option. Could you try the Branch from #22 and use the following command to force rebuild the containers?

```
docker-compose --project-name ddns up -d --build --force-recreate
```

**Comment by xandm** [comment]
Thanks, this has resolved the issue for me :)

**Comment by pboehm** [comment]
I've merged #22 so the fixes are now on master and no specific branch is required anymore. Closing this for now.
